.github/workflows/ci.yml

```diff
@@ -23,9 +23,13 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.0'
-          - '1.3'
+          - '1.6'
+          # - '1.7'
+          # - '1.8'
+          # - '1.9'
           - '1'
+          - '~1.10.0-0'
+          # - 'nightly'
```

Comment:
IMO we should test the stable Julia version (1.9 currently, `1` in CI), the latest LTS (`1.6`), and the oldest supported version (`'1.0'`) to avoid accidentally breaking anything.

We could add nightly but I assume it would be quite noisy - nightly tends to break things quite often (in particular when relying on internals such as LinearAlgebra here) but most of these broken nightlies won't result in a broken stable release in my experience.